### PR TITLE
WRR-11077: Fixed Scroller to prevent the native scrolling behavior caused by keydown events when the popup is opened

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
In this case, the Scroller has a button that opens FixedPopupPanel. The issue is that when press the down key at the moment the popup is opened, the scroller behind the FixedPopupPanel scrolls down.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
To fix this issue, we have to prevent the native scrolling behavior caused by keydown events. 
So I fixed keydown handler of scrollerContainer to call `preventDefault` when a keydown event is caused by a directional key and the spotlight is paused.
To not to have `spotlight` dependency, the preventing function resides in sandstone and enact/Scroller just calls the funcion passed by prop like other cases.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRR-11077

### CODE REVIEW

File: packages/ui/package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
   },
   "dependencies": {
     "@enact/core": "^4.9.4",
+    "@enact/spotlight": "^4.9.4",
```
Comment (MikyungKim) [inline packages/ui/package.json:36]: Please revert this change.

Comment (juwonjeong) [reply packages/ui/package.json]: It's removed

File: packages/ui/useScroll/useScroll.js [hunk 2]
```diff
@@ -132,6 +132,7 @@ const useScrollBase = (props) => {
 	delete rest.onScrollStart;
 	delete rest.onScrollStop;
 	delete rest.onWheel;
+	delete rest.preventScroll;
```
Comment (MikyungKim) [inline packages/ui/useScroll/useScroll.js:135]:
```suggestion
	delete rest.preventScroll; // scrollMode 'native'
```

Comment (juwonjeong) [reply packages/ui/useScroll/useScroll.js]: It's fixed.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by 0x64 — APPROVED** [review]
